seqr/views/utils/orm_to_json_utils.py

```diff
@@ -165,6 +165,7 @@ def _process_result(result, project):
             'isMmeEnabled': result['isMmeEnabled'] and not result['isDemo'],
             'canEdit': has_project_permissions(project, user, can_edit=True),
             'userIsCreator': project.created_by == user,
+            'isAnalystProject': any(c.name == ANALYST_PROJECT_CATEGORY for c in project.projectcategory_set.all())
```

Comment:
in order to prevent this having a performance impact, we need to change the prefetch behavior below to always run  `prefetch_related_objects(projects, 'projectcategory_set')` (currently it is only conditionally run)

Reply:
The condition for `prefetch_related_objects(projects, 'projectcategory_set'` is removed.